Xamarin.Forms.Platform.Android/Renderers/SwipeViewRenderer.cs

```diff
@@ -225,7 +230,7 @@ public override bool OnTouchEvent(MotionEvent e)
 			float x = Math.Abs((_downX - e.GetX()) / density);
 			float y = Math.Abs((_downY - e.GetY()) / density);
 
-			if (e.Action != MotionEventActions.Move | (x > 10f || y > 10f))
+			if (e.Action != MotionEventActions.Move | (x > SwipeMinimumDelta || y > SwipeMinimumDelta))
```

Comment:
I think we should check swipe direction
If left or right & x>delta
If top or bottom & y> delta